src/aks-preview/azcli-aks-live-test/clone_repo.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -eux
+pwd
+
+# clone azure-cli (default is the official repo)
+# git clone https://github.com/Azure/azure-cli.git
+git clone $CLI_REPO
+
+# ckeckout to a specific azure-cli branch (default is the dev branch)
+cd azure-cli/
+git branch -a
+git checkout $CLI_BRANCH
+cd -
+
+# clone azure-cli-extensions when manually specify the extension repo
+if [[ $MANUAL_EXT == true && -n $EXT_REPO && -n $EXT_BRANCH ]]; then
+    echo "Manually specify the extension repo, delete the current 'azure-cli-extensions' directory!"
+    rm -rf azure-cli-extensions/
+    git clone $EXT_REPO
+    cd azure-cli-extensions/
+    git checkout $EXT_BRANCH
+    cd -
+fi
+
+# check current branch & commit logs in azure-cli-extensions
+cd azure-cli-extensions/
```

Comment:
pushd
popd

Reply:
Fixed in [930ca68](https://github.com/Azure/azure-cli-extensions/pull/3292/commits/930ca681a709373757cb8d60998e33d705be1aff).